moped-editor/src/utils/activityLogFormatters/mopedPersonnelActivity.js

```diff
@@ -41,12 +41,24 @@ export const formatPersonnelActivity = (change, userList) => {
     };
   }
 
-  // currently 'notes' is the only other editable field on this table
-  return {
+  // update the notes field
+  if (change.description[0].field === "notes")
+    return {
     changeIcon,
     changeText: [
       { text: "Updated team member notes for " },
       { text: userList[changeData.new.user_id], style: "boldText" },
     ],
   };
+
+  // the only other change that would be reflected here is adding or removing roles
```

Comment:
thanks for adding the extra condition for notes updates that we can check by the field changed. 

I'm keeping in mind that this catch-all for detecting role change could create a future bug if we ever add a new field to personnel records, but I think that chance is low.

Reply:
thanks!